static/js/components/sidebar.js

```diff
@@ -19,7 +19,7 @@ export default class Sidebar extends React.Component {
                   <div>
                     <ul className="Sidebar__filter__list">
                       <li>
-                        <input type="checkbox" id="filter--academic" defaultChecked={this.props.tags['academic_affairs']} onClick={this.props.handleClick} value={"academic_affairs"} />
+                        <input type="checkbox" id="filter--academic" checked={this.props.tags['academic_affairs']} onClick={this.props.handleClick} value={"academic_affairs"} />
```

Comment:
Is this the way it's done these days? I always thought it was `defaultChecked` but that was back when I started learning react. Apparently now you can keep the `class` attribute and not have to write `className` instead... wild haha

Reply:
AFAIK, `defaultChecked` initializes the value but does not control it (i.e. updating the state does not change it, and the user is free to check and uncheck the box to their heart's content and the box's appearance will change as expected). `checked` ensures that the component state will always match the value given to `checked`. See https://reactjs.org/docs/forms.html#handling-multiple-inputs

And Idk about `class` vs `className` -- my IDE still complains if I just use `class`.